sound/soc/sof/ipc4-topology.c

```diff
@@ -890,7 +890,7 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 	}
 
 	cfg_size = sizeof(struct sof_ipc4_base_module_cfg);
-	if (process->payload_with_output_fmt)
+	if (process->init_payload_format & INIT_PAYLOAD_WITH_OUTPUT_FMT)
```

Comment:
more explicitly if (process->init_payload_format == INIT_PAYLOAD_WITH_OUTPUT_FMT)

Reply:
I think that we will have process modules that have reference format and output format at the same time, but WHM(low power WoV) is the only one, and we don't support it in Linux. == test is good. updated.

Reply:
If  init_payload_format will be exactly ONE format, please use enum instead of `#define INIT_PAYLOAD_WITH_OUTPUT_FMT	BIT(0)`

Reply:
Each format takes a single bit, I think macro here is easier to extend. while with enum, you will have specify value for each enum member with 1,2,4,8...